packs/data/pathfinder-bestiary-2.db/culdewen.json

```diff
@@ -682,7 +682,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>A creature struck by the culdewen's fish hook is skewered, taking @Localize[PF2E.PersistentDamage.Bleed1d4.success] as long as the hook remains stuck in it. The fish hook can be removed if a creature @Compendium[pf2e.actionspf2e.Escape]{Escapes} (DC 25), pulling it free.</p>"
+                    "value": "<p>A creature struck by the culdewen's fish hook is skewered, taking @Localize[PF2E.PersistentDamage.Bleed1d4.success] as long as the hook remains stuck in it. The fish hook can be removed if a creature @Compendium[pf2e.actionspf2e.Escape]{Escapes} (@Check[type:athletics|dc:25]), pulling it free.</p>"
```

Comment:
Escape is not necessarily Athletics

Reply:
It is in this specific one. Check the source.